internal/cmd/root.go

```diff
@@ -19,7 +19,11 @@ import (
 	"go.k6.io/k6/cmd/state"
 	"go.k6.io/k6/errext"
 	"go.k6.io/k6/errext/exitcodes"
+	"go.k6.io/k6/ext"
 	"go.k6.io/k6/internal/log"
+	"go.k6.io/k6/secretsource"
+
+	_ "go.k6.io/k6/internal/secretsource" // import it to register internal secret sources
```

Comment:
This is contrast with how it is done for outputs or js modules. 

I kind of prefer this to having a function with all of them in one place which gets constant conflicts.

But I can redo it to be more aligned with other parts of k6.